astropy/stats/lombscargle/core.py

```diff
@@ -50,6 +51,9 @@ class LombScargle(object):
         of the input data. This is especially important if fit_mean = False
     nterms : int (optional, default=1)
         number of terms to use in the Fourier fit
+    normalization : string (optional, default='standard')
```

Comment:
For a fixed list of possible values, the numpydoc style is to list them in braces with the default value first, i.e.:

`normalization : {'standard', 'model', 'log', 'psd'}, optional